spark/src/main/scala/org/apache/spark/sql/comet/operators.scala

```diff
@@ -276,13 +279,30 @@ abstract class CometNativeExec extends CometExec {
     }
   }
 
+  /**
+   * Maps through product elements except the last one. The last element will be transformed using
```

Comment:
nit: Instead of "Maps through product elements .." maybe "Copies product elements to the output array except the last one"?